client/src/components/Button/index.jsx

```diff
@@ -1,9 +1,13 @@
 import { Button as BasicButton } from 'reactstrap';
 import styled from 'styled-components';
 
-const Button = styled(BasicButton)`
+export const Button = styled(BasicButton)`
   background-color: ${({theme}) => theme.color.primary}
   border-color: ${({theme}) => theme.color.primary}
 `;
 
-export default Button;
\ No newline at end of file
+export const CourseTitleButton = styled(Button)`
+  height: ${({theme}) => theme.unit.large}px;
+`
+
+export default Button;
```

Comment:
if Button is already default exported, it doe snot need to be exported again when initializing